<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveSeg</class>
 <widget class="qMRMLWidget" name="InteractiveSeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>889</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="qMRMLSubjectHierarchyTreeView" name="SubjectHierarchyTreeView">
     <property name="levelFilter">
      <stringlist notr="true"/>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="hideChildNodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="includeItemAttributeNamesFilter">
      <stringlist notr="true"/>
     </property>
     <property name="includeNodeAttributeNamesFilter">
      <stringlist notr="true"/>
     </property>
     <property name="excludeItemAttributeNamesFilter">
      <stringlist notr="true"/>
     </property>
     <property name="excludeNodeAttributeNamesFilter">
      <stringlist notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="segmentationCollapsibleButton">
     <property name="text">
      <string>Segmentation</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight">
      <number>9</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select pre-trained segmentation model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="applyButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="editSegmentation">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Edit Segmentation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
